examples/multi-cluster-fleet/python/agent_sandbox_fleet/fleet_member.py

```
@@ -0,0 +1,752 @@
+        # apiserver, competing for the very APF seats a density run measures.
+        # `light` reports warmpool depth/ready only, which is what the
+        # capacity-aware planner leans on anyway.
+        if self.capacity_detail == "light":
+            return report
+
+        # active_claims — via SDK. NOTE: list_all_sandboxes() is unpaginated,
+        # so this is the remaining O(cluster) allocation in `full` mode.
+        try:
+            claims = self.sandbox_client.list_all_sandboxes(namespace=self.namespace)
+            report.active_claims = len(claims)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)